deny.toml

```diff
@@ -34,18 +34,22 @@ highlight = "all"
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
```

Comment:
I don't believe they're explicitly caused by Palette, and might just be because I built this PR on my Windows machine. If you (or anyone else) is able to remove these deny skips I think it should be entirely possible. From what I can see, these aren't actually attached to the palette dependency graph, but I might just be incorrect on that.

Reply:
if they're not from changes in this pr, then you should probably remove them. The check bans ci job is to make sure new dependencies aren't introducing conflicts, but it's also triggered when some dependencies get updates. You shouldn't try and fix the second type.

Reply:
I'll revert this change and double-check that Palette isn't the cause of my local CI issues. Thanks for the clarification!